prime-dotnet-webapi/Controllers/SatEnrolmentController.cs

```diff
@@ -195,6 +199,7 @@ public async Task<ActionResult> SubmitSatEnrollee(int satId)
 
             var existsInPlr = await _plrProviderService.PartyExistsInPlrWithCollegeIdAndNameAsync(satId);
             var submission = await _partyService.CreateSubmissionAsync(satId, SubmissionType.SatEnrollee, existsInPlr);
+            await _keycloakClient.AssignRealmRole(User.GetPrimeUserId(), Roles.PhsaEformsSat);
```

Comment:
We could also make a wrapper for this method in `PhsaService` if we don't want to call the KeyCloak service in the controller